### ISSUE TITLE
Identify how user will determine if they get SSI or SSDI

### ISSUE THREAD

**Issue by twhiles** [nonprofittechy/docassemble-ssareportchangesletter#44]
Most laypersons don't know how to find this information at a glance. Need to provide guidance is this interview is to be completed by recipient, versus 3rd party (i.e. attorney)

**Comment by twhiles** [comment]
This was opened with title "bug report" below - issue #21; will leave both open for now.

**Comment by Landon-Shimel** [comment]
I agree that this is something most laypersons struggle with. I did some research and here are some links that might be helpful to include on that page: 

https://www.ncoa.org/article/ssi-vs-ssdi-what-are-these-benefits-how-they-differ/

https://www.usa.gov/social-security-disability

https://soarworks.samhsa.gov/article/overview-of-social-security-disability-programs-ssi-and-ssdi

**Comment by twhiles** [comment]
added hyperlink successfully, but running into another (unrelated) error. Will push code once it is working again.
